libgadget/winds.c

```diff
@@ -254,61 +286,55 @@ winds_evolve(int i, double a3inv, double hubble)
 static void
 sfr_wind_weight_postprocess(const int i, TreeWalk * tw)
 {
+    int done = 0;
     if(P[i].Type != 4)
         endrun(23, "Wind called on something not a star particle: (i=%d, t=%d, id = %ld)\n", i, P[i].Type, P[i].ID);
-    int diff = WINDP(i).Ngb - 40;
+    struct winddata * Windd = WIND_GET_PRIV(tw)->Winddata;
```

Comment:
CamelCases tends to invoke imaginations of global variables.

As a local var name it can be confusing, but this trend started with the config variables.